scripts/eval/eval.py

```diff
@@ -133,6 +133,11 @@ def evaluate_model(model_cfg: DictConfig, dist_timeout: Union[float, int],
             [t.name for t in eval_gauntlet_callback.categories])
 
     load_path = model_cfg.get('load_path', None)
+    if model_cfg.model.name == 'mpt_causal_lm' and load_path is None:
+        raise ValueError(
+            'MPT causal LMs require a load_path to the checkpoint for model evaluation. \
```

Comment:
this results in a biiiiig space in the middle of the string :) you can just do multiline strings with + in between. should be examples in other places in the codebase

Reply:
thanks, fixed 👍